src/app/components/tree-view/index.html

```diff
@@ -74,7 +78,7 @@ <h3>Selection</h3>
       <h3>Split view</h3>
```

Comment:
Just my 2 cents, but the split view example doesn’t seem like an option to me. It seems like a use case that should be in the Use when section, not an option for this section. I mean, this isn't an option you select on the tree view components. It's an implementation of the tree view components inside of another component.

Reply:
I think the issue is the header saying "Split view" instead of "Active mode" which is intended for use specifically with split views.

Reply:
Maybe I'm just confused here, but this still doesn't seem like an option to me. I mean, isn't this just default behavior? If the tree view is included in a split view, then isn't "active mode" just the the indicator that the component implements automatically when users select an item because the tree view isn't in a select mode? I may be splitting hairs here, but I feel like I'm missing something.

Reply:
The issue is that the 3rd party library has wonky states/modes. From a UX pattern perspective, "Active mode" is a thing regardless of how it's implemented.

Reply:
OK. So should we change the header to "Active mode" and update the text to spell out what active mode is and how it pertains to split views?

Reply:
yeah

Reply:
I changed the header to "Active state" and fleshed the paragraph out a bit to try to emphasize that as the option.